firmware/quadruna/BMS/src/app/states/app_allStates.c

```diff
@@ -144,6 +145,12 @@ bool app_allStates_runOnTick100Hz(void)
     app_airs_broadcast();
     app_shdnLoop_broadcast();
 
+    bool diagnostics_mode_enabled = app_canRx_Debug_CellDiagnosticsRequest_get();
```

Comment:
```suggestion
    bool diagnostics_mode_enabled = app_canRx_Debug_CellDiagnosticsRequest_get();
```

This variable isn't bool in the Debug_tx.json anymore (its an enum rn)